#!/user/bin/env python3
CAMERA_TOPIC = "/camera/image"

RIGHT_REAR_TOPIC = "/racecar/right_rear_controller/command"
LEFT_REAR_TOPIC = "/racecar/left_rear_controller/command"
RIGHT_FRONT_TOPIC = "/racecar/right_front_controller/command"
LEFT_FRONT_TOPIC = "/racecar/left_front_controller/command"

LEFT_DIRECTION_TOPIC = "/racecar/left_direction_controller/command"
RIGHT_DIRECTION_TOPIC = "/racecar/right_direction_controller/command"